api/src/test/java/org/openmrs/ObsTest.java

```diff
@@ -1032,7 +1032,7 @@ public void newInstance_shouldCopyMostFields() throws Exception {
         obs.setDateVoided(new Date());                                
         obs.setVoidReason("Some reason");    
         obs.setValueComplex("SDIODF7980IDSISF");
-        obs.setComplexData(new ComplexData("Complex data", "Complex data".getBytes()))
```

Comment:
Is this change part of the ticket?

Reply:
Sorry for the delay but the changes are part of the ticket TODO as described TODO in this link https://github.com/openmrs/openmrs-core/pull/2811/commits/e0929d880cec681056b1b45a3b9683a766ffac5e#diff-30732ed0378eb70e0c9eb7625ab3af04L1030